doc/connectivity/bluetooth/api/mesh/access.rst

```diff
@@ -185,17 +185,24 @@ information about the device. In order to access this information, the user
 may use the :ref:`bluetooth_mesh_models_cfg_srv` model or, if supported,
 the :ref:`bluetooth_mesh_lcd_srv` model.
 
-Composition Data Page 0 and 128
--------------------------------
+Composition Data Page 128, 129 and 130
```

Comment:
Wouldn't it be more appropriate to push this section down and after CDP2? It reads bit funny otherwise.

Reply:
Oh wow, yes I of course agree. The order made sense in a previous revision, but does not make sense anymore. I'll move it.